### PR TITLE
[375] Add license file

### PR DESCRIPTION
**Pull request by arturtamborski**
Solves #375 
Please review and approve.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Code For Poznań (codeforpoznan.pl)
```
Comment (arturtamborski) [inline LICENSE]:
```suggestion
Copyright (c) 2020 Code For Poznań (codeforpoznan.pl) and contributors.
```

**Comment by magul** [comment]
I will personally try to communicate with all of previous contributors to ensure all are ok with this license applied.

Give me, please, a few weeks to do so.

**Comment by OtisRed** [comment]
Has there been any progress in that regard? Not having a license could be actually a big deal formally, because without it, since we are not PAH volunteers per se, using the app by PAH might constitute the unjustified income. Since there is no other formal agreement between them and us (because "us" don't legally exist), having a license would prevent that problem. I suggest to add one as quick as possible.

@magul I remember we agreed to talk to each contributor and give one a chance to agree. If it's been done and the rest just didn't respond I would take it as a silent consent. If you didn't contact them I would merge it anyway - we all know what we are doing and even if some contributors didn't express that explicitly the implicit working agreement is still binding. I think it's highly unlikely that anyone would litigate and even if somebody were to, the case in that regard seems very frail. Meanwhile from the formal standpoint it seems more pressing to formalize the use of software so that PAH would not be at risk (@OlaPAH please correct me if that's for some reason an irrational panic on my part ^^).

**Comment by arturtamborski** [comment]
bump @OtisRed, @magul, @monikakrawczak any decision on it?

**Pull request merged**